hyperdome_client/__init__.py

```diff
@@ -38,15 +38,17 @@ class Application(QtWidgets.QApplication):
     """
 
     def __init__(self, common):
-        if common.platform == 'Linux' or common.platform == 'BSD':
+        if common.platform == "Linux" or common.platform == "BSD":
```

Comment:
You ignored the refactor suggestion? :o

Reply:
I think there's enough suggestions for a wider-scale refactor and I didn't want to get too in the weeds on it